src/client/pages/StaffViewAuthPage/index.test.js

```diff
@@ -0,0 +1,27 @@
+import renderNonTransContent from "../../test-helpers/renderNonTransContent";
+import AUTH_STRINGS from "../../../data/authStrings";
+import Component from "./index";
+
+describe("<StaffViewAuthPage />", () => {
+  it("retro certs auth page", async () => {
+    const wrapper = renderNonTransContent(Component, "StaffViewAuthPage");
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("retro certs auth page with user not found error", async () => {
+    const wrapper = renderNonTransContent(Component, "StaffViewAuthPage", {
+      userData: { status: AUTH_STRINGS.statusCode.userNotFound },
+    });
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("retro certs auth page with session timeout", async () => {
```

Comment:
Can we remove this test? This error message shouldn't show up.

Reply:
oh right, good point!